reference/orders.v2.oas2.yml

```diff
@@ -693,7 +693,7 @@ paths:
 
         Acceptable values for `shipping_provider` include an empty string (`""`), auspost, canadapost, endicia, usps, fedex, royalmail, ups, upsready, upsonline, or shipperhq.
 
-        Acceptable values for `tracking_carrier` include an empty string (`""`) or one of the valid tracking-carrier values viewable [here](https://github.com/bigcommerce/dev-docs/blob/master/assets/csv/tracking_carrier_values.csv) and downloadable as a .CSV file [here](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/csv/tracking_carrier_values.csv).
+        Acceptable values for `tracking_carrier` include an empty string (`""`) or one of the valid [tracking-carrier values](https://github.com/bigcommerce/dev-docs/blob/master/assets/csv/tracking_carrier_values.csv) and the [downloadable .CSV file](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/csv/tracking_carrier_values.csv).
```

Comment:
solve these issues:
* this sentence doesn't scan -- look at it without the markdown links.
* the raw version in github can easily be copied and pasted, but it is not a download, so that's misleading
* this is two links to the same content

Reply:
Relevant slack conversation [here](https://bigcommerce.slack.com/archives/C039JSWHH50/p1670865268099679?thread_ts=1670848230.004619&cid=C039JSWHH50)